aepp/flowservice.py

```diff
@@ -543,19 +542,29 @@ def createFlow(
                 "transformations": [],
                 "scheduleParams": {}
             }
-            if kwargs.get("export_mode",None) is not None:## infer dataset export
-                obj["scheduleParams"]["exportMode"] = kwargs.get("export_mode",None)
-                obj["scheduleParams"]["foldernameTemplate"] = "%DESTINATION%_%DATASET_ID%_%DATETIME(YYYYMMdd_HHmmss)%"
-            if kwargs.get("schedule_timeUnit",None) is not None:
-                obj["scheduleParams"]["timeUnit"] = kwargs.get("schedule_timeUnit",None)
+            if schedule_frequency == "once":
+                obj["scheduleParams"] = {
+                    "timeUnit": "day",
+                    "interval": 0,
+                    "exportMode": "DAILY_FULL_EXPORT"
+                }
+            elif schedule_frequency not in ("minute", "hour"): 
```

Comment:
don't you want to have 
`schedule_frequency in ("minute", "hour"):` (remove the not)